### PR TITLE
Add Bazelmod BCR configuration

### PR DESCRIPTION
**Pull request by Bencodes**
Adding Bazelmod BCR configurations so that future releases can be deployed to the BCR.

### CODE REVIEW

**Comment by Bencodes** [comment]
@utzcoz are you able to approve the [Public to BCR](https://github.com/apps/publish-to-bcr/) app that I requested for this repo?

File: .bcr/source.template.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "integrity": "sha256",
```
Comment (keith) [inline .bcr/source.template.json:2]:
```suggestion
  "integrity": "",
```

**Comment by utzcoz** [comment]
@Bencodes Sorry, I don't receive installation request of this app from you. But I installed it to the robolectric-bazel repository now. Maybe you can check it. cc @hoisie @brettchabot for extra GitHub app installed into robolectric-bazel repository with read and write access.

File: .bcr/config.yml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 fixedReleaser:
   login: Bencodes
-  email: ben@ben.cm
\ No newline at end of file
+  email: ben@ben.cm
```
Comment (utzcoz) [inline .bcr/config.yml:3]: Can we use git attributes to format the EOL?

File: .bcr/metadata.template.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "homepage": "https://github.com/robolectric/robolectric-bazel",
+  "maintainers": [
```
Comment (Bencodes) [inline .bcr/metadata.template.json:3]: @utzcoz i'm going to land this, but feel free to open up additional PRs adding extra maintainers.

Comment (utzcoz) [reply .bcr/metadata.template.json:3]: Thanks for your awesome work.

**Pull request merged**